app/components/article-featured-video.js

```diff
@@ -23,7 +25,37 @@ export default Component.extend(
 		didRender() {
 			this._super(...arguments);
 
-			this.initVideoPlayer();
+			if (!this.get('hasRendered')) {
```

Comment:
This is misleading. How could it `didRender` but not `hasRendered`? Is the purpose to init the video player only on the first render? If so, the `didInsertElement` hook is called only once, see https://guides.emberjs.com/v2.12.0/components/the-component-lifecycle/#toc_integrating-with-third-party-libraries-with-code-didinsertelement-code